Source/Core/Core/ConfigManager.cpp

```diff
@@ -51,6 +52,12 @@ SConfig::SConfig()
   LoadDefaults();
   // Make sure we have log manager
   LoadSettings();
+  Config::AddConfigChangedCallback([this] {
```

Comment:
Note that callbacks added to `AddConfigChangedCallback` are (currently) never deleted. This means that calling `SConfig::Shutdown()` followed by `SConfig::Init()` would result in two of these, one of which with a dangling `this` pointer.